src/concern/mod.rs

```diff
@@ -152,8 +175,9 @@ impl Acknowledgment {
 }
 
 impl WriteConcern {
+    #[allow(dead_code)]
     pub(crate) fn is_acknowledged(&self) -> bool {
-        self.w != Some(Acknowledgment::Nodes(0))
+        self.w != Some(Acknowledgment::Nodes(0)) && self.journal != Some(false)
```

Comment:
Should this be `||` rather than `&&`? `w=1, j=false` is acknowledged, but this expression would yield false

Reply:
Whoops, fixed. Also added a test.

Reply:
I'll be honest, I had to stare at this line for almost a full minute to convince myself that it did in fact need to be `||`. Once "not" gets involved, it's very hard to keep everything straight in your head.